temporal-sdk/src/main/java/io/temporal/client/WorkflowClient.java

```diff
@@ -741,6 +742,13 @@ static <A1, A2, A3, A4, A5, A6, R> CompletableFuture<R> execute(
       A4 arg4,
       A5 arg5,
       A6 arg6) {
-    return WorkflowClientInternal.execute(workflow, arg1, arg2, arg3, arg4, arg5, arg6);
+    return WorkflowClientInternalImpl.execute(workflow, arg1, arg2, arg3, arg4, arg5, arg6);
   }
+
+  /**
+   * For SDK Internal usage only. This method should be used by users. If implementing a proxy or an
```

Comment:
```suggestion
   * For SDK Internal usage only. This method should not be used by users. If implementing a proxy or an
```

Reply:
I'd even suggest putting the `not` in bold.